framework/log/FileTarget.php

```diff
@@ -105,6 +101,11 @@ public function init()
      */
     public function export()
     {
+        $logPath = dirname($this->logFile);
+        if (!is_dir($logPath)) {
+            FileHelper::createDirectory($logPath, $this->dirMode, true);
+        }
```

Comment:
`export()` can be triggered multiple times in the application flow, its probably better to do it in `init()` but only if `enabled` is true.

Reply:
thinking more about it, my suggestion will make it much more complex as the path should still be created after enabling a disabled log target. Looks good to me as is.